### DISCUSSION TITLE
Ignore files by mtime

### DISCUSSION THREAD

**Discussion by adenu** [paulmillr/chokidar#1340] [Q&A]
Is it possible to ignore files on the watcher based on their modified date? Im trying to implement chokidar to watch for recent invoices (90 days) in a folder, but when the folder has like 3k+ files (because it includes invoices even from 2023, 2022) the lower spec systems goes bananas. And this happens on every startup of the app. Have anyone gone through something like that? 

(one system had 100k+ files, the owner uninstalled the app immediately since it froze everything.)

**Comment by paulmillr** [comment]
use `ignored` option:

`ignored: (path, stats) => // do things with stats.mtime`
